src/Goteo/Controller/InvestController.php

```diff
@@ -91,8 +91,17 @@ private function validate(
 
         $custom_amount = Currency::amountInverse($amount_original, $currency);
         $project_categories = Project\Category::getNames($project_id);
+        $donate_amount = Currency::amount(
+            Config::get('donate.tip_amount'),
+            Currency::get(Config::get('currency'))
```

Comment:
The execution of these lines throws an error since the Currency::get() function returns an array, that the Currency::amount() is not expecting.